.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/*
```

Comment:
Please use our standard gitignore file, you can see it at various other plugins.